### PR TITLE
This PR includes:

### LINKED ISSUE #31: Step 11
## Value Proposition
As a user, I want to see all selections that I had made in favourite articles, and pin them in one or more board
## Description
description_step11

### On the dashboard page user should be able to: 
- [ ] see all the selections they had made in their favourite articles
- [ ] drag any selection to a board
## Tasks
- [ ] update articles with selections to database
- [ ] load all selections from database to the draggable section in dashboard page
- [ ] add a save icon to the dropzone section
- [ ] upon clicking the save icon:
- [ ] [ ] the dashboard will be posted to database
- [ ] [ ] the dropzone is cleared
- [ ] add a link "go here to see the dashboards you have saved" to another page where users can see their saved boards

### PR DESCRIPTION
**Pull request by anhthu253**
- save selections being made in favourite articles to database
- load all selections to dashboard page
- save dashboards to DB
- load all dashboards from DB to an extra page

### CODE REVIEW

File: components/Dropzone.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 
-export default function Dropzone({ id, ondragover, ondrop, children }) {
+export default function Dropzone({ id, ref, ondragover, ondrop, children }) {
```
Comment (Kheirah) [inline components/Dropzone.js]: How is the ref prop being used in this component?

File: pages/api/dashboard/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import dbConnect from "../../../lib/dbConnect";
+import DashBoard from "../../../models/DashBoard";
+import FavouriteArticle from "../../../models/FavouriteArticle";
+import { getAllDashBoards } from "../../../services/dashboardService";
```
Comment (Kheirah) [inline pages/api/dashboard/index.js]: You may remove unused imports.

File: pages/content.js [hunk 3]
```diff
@@ -20,6 +20,7 @@ export async function getServerSideProps() {
 export default function Content({ currentCollections }) {
   const currentArticle = useStore((state) => state.currentArticle);
   const [popUp, setPopUp] = useState(false);
+  const [selections, setSelections] = useState([]);
   const contentRef = useRef();
```
Comment (Kheirah) [inline pages/content.js:24]: I can't see how a reference to your `StyledContent` component is useful to you in this page. If you're not using it, consider removing it.

**Review by Kheirah — APPROVED** [review]
Very good!

Just have a look at the comments :)

File: pages/api/dashboard/index.js (hunk 2 above)
Comment (anhthu253) [reply pages/api/dashboard/index.js]: yes indeed 🙈

File: pages/content.js (hunk 3 above)
Comment (anhthu253) [reply pages/content.js:24]: I used this to refer to the content updated by being highlighted or by removing highlights. Could u not see the whole file content.js in this PR? I m very bad with github and suck at deciding what should belong to which PR.. 😕

**Comment by anhthu253** [comment]
Thank you a lot Ernst! :)

File: components/Dropzone.js (hunk 1 above)
Comment (anhthu253) [reply components/Dropzone.js]: opps i forgot to remove it 😊

**Comment by anhthu253** [comment]
oh I just saw that I forgot to delete the dummy data too 😬

File: pages/content.js (hunk 3 above)
Comment (Kheirah) [reply pages/content.js:24]: Ye, sometimes it's difficult to see the overall picture when code is collapsed. If you are indeed using `contentRef`, then it's ok :)

**Pull request merged**